### DISCUSSION TITLE
CodeTracker: Accidently Moved incomplete assignment card from In Progress to Done

### DISCUSSION THREAD

**Discussion by jimguistical** [nss-evening-cohort-14/evening-client-side#37] [Help Me!] [answered]
Jim Conner

Searched other help tickets -- yes 
What have you Googled? n/a

Describe your Issue:
I accidently moved the Product Cards assignment card from In Progress to Done last night (meant to move the course feedback week 3 assignment instead).  It doesn't look like moving cards to Done _directly_ impacts our work in repos... that is, looks like I can still work on project even after moving the card to "Done."  I don't want the Instruction Team to review my project though and think it's done!

What is best practice for alerting the team that I am not actually done with the assignment?  Or can I just continue working on the project and not need to edit/request a card to be moved back to "In Progress" in CodeTracker.  Thanks!

**Comment by drteresavasquez** [comment, marked as answer]
You can send a slack message for codeTracker related issues to the whole instruction team. I can go ahead and move it back for you 😊
